openpype/hosts/flame/plugins/publish/extract_subset_resources.py

```diff
@@ -90,38 +90,38 @@ def process(self, instance):
         handle_end = instance.data["handleEnd"]
         handles = max(handle_start, handle_end)
         include_handles = instance.data.get("includeHandles")
-        retimed_handles = instance.data.get("retimedHandles")
+        not_retimed_handles = instance.data.get("notRetimedHandles")
```

Comment:
Out of interest - what's the reason this logic is inversed.
The setting is **Retime Handles** but we're storing and using **Not Retime Handles**?

I feel it's slightly adding to the complexity of the code by having that inverted.

Reply:
Yes, I agree. The only reason is backwards compatible with all hosts. But perhaps there is a better way of doing this. I will change this.

Reply:
Check it now please if it makes more sense @BigRoy